subcommands/keys/resign_root.go

```diff
@@ -16,11 +16,14 @@ var dryRun bool
 
 func init() {
 	resign := &cobra.Command{
-		Use:   "resign-root <offline key archive>",
-		Short: "Re-sign the Factory's TUF root metadata",
-		Run:   doResignRoot,
-		Args:  cobra.ExactArgs(1),
-		Long:  "The root metadata, root.json expires. Re-signing bumps the expiriation date.",
+		Use:    "resign-root <offline key archive>",
+		Short:  "[DEPRECATED] Re-sign the Factory's TUF root metadata",
+		Hidden: true,
+		Run:    doResignRoot,
+		Args:   cobra.ExactArgs(1),
+		Long: `Please, use a more secure way to keep your root role fresh - rotate the root key.
+
+Ability to refresh expiration date using the resign-root will be removed in upcoming releases...`,
```

Comment:
Why not remove it now?

Reply:
There may be customer scripts which still use it; a good habit is to produce the deprecation warning for some reasonable period like e.g. 3-6 months.